lib/voyage/app_builder.rb

```diff
@@ -481,11 +481,11 @@ def customize_user_factory(adding_first_and_last_name)
       inject_into_file 'spec/factories/users.rb', before: /^  end/ do <<-'RUBY'.gsub(/^ {4}/, '')
         password 'asdfjkl123'
         password_confirmation 'asdfjkl123'
-        sequence(:email) { |n| "user_#{n}@example.com" }
+        email { "user_#{uuid}@example.com" }
```

Comment:
This will make logging in hard, as I believe we prime using these, so we get user_1@example.com

Reply:
Though it looks like you're manually creating an admin below, so maybe that's not a big deal.

Reply:
I'd really rather not use the UUID at all for these factories, but the current test setup doesn't appear to properly reset the database between each test. Maybe if we integrate / setup DatabaseCleaner that would obviate any need for this.

Reply:
Is it hitting an issue creating a user again? I would have thought the sequence would have been consecutive over the test run

Reply:
I would have thought so too, but something else appears to be at play here. Once we button up these few remaining issues, I hope to go back and review the specs to get everything passing successfully and bump the coverage if there's time as well.